src/test/java/POM/FactoryPages/HomePage.java

```diff
@@ -0,0 +1,34 @@
+package POM.FactoryPages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends BasePage{
+
+    @FindBy(css="img.avatar")
+    WebElement avatarLocator;
+    @FindBy(css="[data-testid = 'sound-bar-play']")
+    WebElement soundBarPlayLocator;
+    @FindBy(css="li a.songs")
+    WebElement allSongsMenuItemLocator;
```

Comment:
**I recommend:**  make **all the locators and WebElements private**, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example


```
    @FindBy(css="img.avatar")
    private WebElement avatarLocator;
    @FindBy(css="[data-testid = 'sound-bar-play']")
    private WebElement soundBarPlayLocator;
    @FindBy(css="li a.songs")
    private WebElement allSongsMenuItemLocator;
```